src/features/diary/login/ui/Login.module.scss

```diff
@@ -0,0 +1,5 @@
+._ {
```

Comment:
there is no need to use style files, use markup from ant d

Reply:
use for divs

Reply:
there is no need to use